logging/cluster-logging-upgrading.adoc

```diff
@@ -1,20 +1,31 @@
 :_content-type: ASSEMBLY
 :context: cluster-logging-upgrading
 [id="cluster-logging-upgrading"]
-= Updating OpenShift Logging
+= Updating Logging
```

Comment:
Is there any reason to drop "openshift" here?

Reply:
Because it didn't match what was in the left nav and we're supposed to keep them consistent